gui/src/Console/State.purs

```diff
@@ -12,8 +12,8 @@ import Global.Effects
 
 initialState :: Channel Action -> State
 initialState chan =
-    { buffer : ">"
-    , display : ">"
+    { buffer : ""
+    , display : ""
```

Comment:
This was changed because this is then entered each time after you run a command and the real throws an erroe